data/json/items/tools.json

```diff
@@ -123,6 +123,25 @@
     ],
     "magazine_well": 4
   },
+  {
+    "id": "UPS_off_air",
```

Comment:
Strange id.

Reply:
The air system is strongly influenced by the UPS system and was therefore created alongside it, to make it easier to understand for others and to make future changes easier to apply to both of them. Furthermore, compressed air is used in power supplies IRL:
https://en.wikipedia.org/wiki/Compressed_air_energy_storage

though practical units were when I checked years ago prohibitively expensive.

Reply:
> to make it easier to understand for others

It will confuse others instead as there is no relation between this system and UPS system at all.